jetty-io/src/main/java/org/eclipse/jetty/io/ClientConnectionFactory.java

```diff
@@ -45,4 +46,31 @@ public default Connection customize(Connection connection, Map<String, Object> c
             client.getBeans(Connection.Listener.class).forEach(connection::addListener);
         return connection;
     }
+
+    public static class Info
```

Comment:
oh another `Info` class with a list of protocols?   is there a common base class needed? again is `Info` the right name?

Reply:
Yes, `Info` here is correct. 
Note that `Info` is rarely used - most of the times application will use `ClientConnectionFactoryOverHTTP2.H2C` or `HttpClientConnectionFactory.HTTP` or other more expressive names to configure `HttpClientTransportDynamic`.